src/transformers/data/data_collator.py

```diff
@@ -142,3 +142,39 @@ def mask_tokens(self, inputs: torch.Tensor) -> Tuple[torch.Tensor, torch.Tensor]
 
         # The rest of the time (10% of the time) we keep the masked input tokens unchanged
         return inputs, labels
+
+
+@dataclass
+class DataCollatorForLazyLanguageModeling(DataCollatorForLanguageModeling):
+
+    block_size: int = 512
+
+    def collate_batch(self, examples: List[torch.Tensor]) -> Dict[str, torch.Tensor]:
```

Comment:
Hello 👋

Thanks for the PR! I tried the `DataCollatorForLazyLanguageModeling` and `LazyLineByLineTextDataset` with `transformers==3.0.2`, and somehow I had to rename `collate_batch` to `__call__` to make it work.

Not sure if I'm missing something - dropping a note here in case someone runs into the same issue.

Thanks again!